tests/integration/optimizers/flow.yml

```diff
@@ -5,5 +5,5 @@ env:
   JINA_DUMMYCRAFTER_PARAM2_VAR: ${{JINA_DUMMYCRAFTER_PARAM2}}
   JINA_DUMMYCRAFTER_PARAM3_VAR: ${{JINA_DUMMYCRAFTER_PARAM3}}
 pods:
-  - uses: pods/craft.yml
-  - uses: pods/evaluate.yml
+  - uses: $JINA_OPTIMIZER_CRAFTER_FILE
+  - uses: $JINA_OPTIMIZER_EVALUATOR_FILE
```

Comment:
We don't care about this anywhere else in the codebase, do we? That makes our whole tests sooo much more ugly. If this is the sacrifice for supporting windows, I'd rather not support it @JoanFM

Reply:
yes this is not so frequently used at other places. I can change it back to have it consistent with the rest

Reply:
And in all our examples we never used environment variables for defining pods. I see your problem with the backslash, but the solution shouldn't be environment variables, but rather a during parsing, that the path resolution in ignorant to the operating system.

Reply:
True. If we solve this in a centralised way, we don't have to care about this anymore.